releases/7.6.0.md

```diff
@@ -35,6 +35,7 @@ Arches 7.6.0 Release Notes
 - 9768 Filter out tiles created during resource creation from activity stream API
 - 9769 Ensure resource creation edit log timestamps precede resource update timestamps
 - 10738 Adds Github action for comparing test coverage between branches and rejecting branches that lower test coverage
+- 11211 Allow use of graph slug in ES reindex resources management command
```

Comment:
I think we probably need to target 8.0 for this, the submission deadline was Monday

Reply:
Moved to 8.0